.github/workflows/sonarcloud.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Analyze with SonarCloud
 
         # You can pin the exact commit or the version.
-        uses: SonarSource/sonarcloud-github-action@v3.1.0
+        uses: SonarSource/sonarcloud-github-action@v4.0.0
```

Comment:
This new version of the GitHub action for SonarQube Cloud is going to be the last one specifically designed for SonarQube Cloud.

Since v4.1.0, the former GitHub action for SonarQube Server ([sonarqube-scan-action](https://github.com/SonarSource/sonarqube-scan-action/)) is the entrypoint for both Server and Cloud: a single GitHub action to interact with the SonarQube solution, whether on premise or in the cloud!

You can read more about the unified action [here](https://community.sonarsource.com/t/the-new-release-of-github-action-for-sonarqube-v-4-1-0-server-and-cloud/131399).